refactor/urls.js

```diff
@@ -1,16 +1,28 @@
 // Refactor this code
 
 function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
+  let url = "/tasks";
 
-    if (tasks.nextTasks) {
-        url += '?hasNext=true';
-    }
+  //   let url = dev ? `/tasks?status=ACTIVE&dev=true&size=20` : "/tasks";
```

Comment:
Why is dead code left here?

Reply:
I wanted to show the changes made above the original ones.
I will update it.